rust_src/src/dispnew.rs

```diff
@@ -136,4 +143,65 @@ pub fn internal_show_cursor_p(window: LispWindowOrSelected) -> bool {
     !win.cursor_off_p()
 }
 
+/***********************************************************************
```

Comment:
We have left these comments behind in the past.

Reply:
There are a few more of these in `dispnew.rs`. Should they be removed?

Reply:
I personally like the grouping comments. We have been copying code without worrying about how easy it is to ready. At some point we are going to need to take some time and make the project readable, discoverable, etc.

I am not certain bringing the block descriptors now matters.